packages/form-builder/src/settings/template/index.tsx

```diff
@@ -2,14 +2,19 @@ import {PanelBody, PanelRow, SelectControl} from '@wordpress/components';
 import {__} from '@wordpress/i18n';
 import {setFormSettings, useFormSettings, useFormSettingsDispatch} from '../../stores/form-settings';
 
+declare global {
+    interface Window {
+        storageData?;
```

Comment:
Let's add a type of `FormTemplate` in our types folder that has an ID and Name.  Then within `storageData` here we can add templates with a type of `FormTemplate[]`.  Then we'll get all the type hinting within `templateOptions`

Reply:
Good point - resolved. See https://github.com/impress-org/givewp-next-gen/pull/83/commits/2f3e9c1d0547dee62d56137dafe0934f0e0a6f65